hw/top_englishbreakfast/lint/top_englishbreakfast_dv_lint_cfgs.hjson

```diff
@@ -0,0 +1,142 @@
+// Copyright lowRISC contributors.
```

Comment:
@sriyerg 
i feel like it's worth making this a symlink also long term..by changing the original one to not necessarily hardcode the top name.

What do you think?

Reply:
That would certainly be nice, but I am not sure where the file is used/called. If it's inside dvsim, I will probably need some hand holding.

Reply:
I've discussed this with Michael and we agreed that we shouldn't make the top name a variable here. On one hand it's a config file and not a template. On the other hand we don't need the full lint on the secondary top-levels. Just the IPs that are top-level specific and the top-level. But not the full set of IPs and DV code.

For these reasons, we decided to remove this file.